Include/patchlevel.h

```diff
@@ -8,28 +8,28 @@
 */
 
 /* Values for PY_RELEASE_LEVEL */
-#define PY_RELEASE_LEVEL_ALPHA	0xA
-#define PY_RELEASE_LEVEL_BETA	0xB
-#define PY_RELEASE_LEVEL_GAMMA	0xC     /* For release candidates */
-#define PY_RELEASE_LEVEL_FINAL	0xF	/* Serial should be 0 here */
-					/* Higher for patch releases */
+#define PY_RELEASE_LEVEL_ALPHA  0xA
+#define PY_RELEASE_LEVEL_BETA   0xB
+#define PY_RELEASE_LEVEL_GAMMA  0xC     /* For release candidates */
+#define PY_RELEASE_LEVEL_FINAL  0xF     /* Serial should be 0 here */
+                                        /* Higher for patch releases */
 
 /* Version parsed out into numeric values */
 /*--start constants--*/
-#define PY_MAJOR_VERSION	3
-#define PY_MINOR_VERSION	7
-#define PY_MICRO_VERSION	0
-#define PY_RELEASE_LEVEL	PY_RELEASE_LEVEL_ALPHA
-#define PY_RELEASE_SERIAL	2
+#define PY_MAJOR_VERSION        3
+#define PY_MINOR_VERSION        7
+#define PY_MICRO_VERSION        0
+#define PY_RELEASE_LEVEL        PY_RELEASE_LEVEL_ALPHA
+#define PY_RELEASE_SERIAL       2
 
```

Comment:
Note that the version constants are automatically generated by the release management tool, https://github.com/python/release-tools/blob/master/release.py, so that will probably need to be modified to handle spaces instead of tabs and any changes there need to be able to continue to handle patchlevel.h in older release cycles.

Reply:
Oh, my bad.

I have created a PR for the release management tool: https://github.com/python/release-tools/pull/2 .